system/base/AdminHandler.cfc

```diff
@@ -8,6 +8,9 @@
 		<cfargument name="eventArguments" type="struct" required="true" />
 
 		<cfscript>
+			if( isBot() ){
```

Comment:
The key here really is to check whether or not the request is a "stateless" request, or not. That is controlled in your Application.cfc with:

```super.setupApplication(
    // ...
    , statelessUrlPatterns = [ ... ]
    , statelessUserAgentPatterns = [ ... ]
);```

(or by implementing your own `isStatelessRequest()` method in your Application.cfc). 

You are then able to check whether or not the request is stateless with `event.isStatelessRequest()` so this should be used instead of `isBot()`.